src/main/kotlin/biokotlin/cli/MafToGvcfConverter.kt

```diff
@@ -66,6 +66,9 @@ class MafToGvcfConverter : CliktCommand(help = "Create a GVCF file from a MAF fi
         .flag(default = false)
 
     override fun run() {
+
+        logCommand(this)
+
```

Comment:
Since the command and parameters are now logged, will you remove the println below that starts with "LCJ"?

Reply:
Ok, I removed it